test/rules/experiments/nnbd/rules/avoid_init_to_null.dart

```diff
@@ -18,12 +24,20 @@ class X {
 
   // TODO(pq): ints are not nullable so we'll want to update the lint here
   // since it will produce a compilation error.
-  // int x = null; //LINT
+  int x = null; //OK (compilation error)
+  int? xx = null; //LINT
   int y; //OK
-  int z;
+  int z; //OK
+
+  X({int a: null}) //OK (compilation error)
```

Comment:
Test for nullable parameter?

Reply:
See `{this.xx: null}` below.  Is that what you mean?

Reply:
Ah, you reused fields to also test parameters.
Clever! :-)